File: py2cpp/examples/randint.cpp
#include <fmt/core.h>
#include <cstdlib> // import atoi
#include <experimental/random> // import randint

int main(int argc, const char* argv[])
{
    auto a = std::atoi(argv[1]);
    auto b = std::atoi(argv[2]);
    auto c = std::experimental::randint(a, b);
    fmt::print("The answer is {}.\n", c);
}

File: py2cpp/examples/randint.py
#! /usr/bin/env python3
import sys
from random import randint


def main(args):
    a = int(args[1])
    b = int(args[2])
    c = randint(a, b)
    print("The answer is {}.".format(c))


if __name__ == "__main__":
    main(sys.argv)
